packages/block-editor/src/store/actions.js

```diff
@@ -232,17 +232,24 @@ export function selectBlock( clientId, initialPosition = 0 ) {
 
 /**
  * Yields action objects used in signalling that the block preceding the given
- * clientId should be selected.
+ * clientId (or optionally, its first parent from bottom to top)
+ * should be selected.
  *
- * @param {string} clientId Block client ID.
+ * @param {string}  clientId      Block client ID.
+ * @param {boolean} orFirstParent If true, select the first parent if there is no previous block.
  */
 export const selectPreviousBlock =
-	( clientId ) =>
+	( clientId, orFirstParent = false ) =>
```

Comment:
@draganescu can you create a small follow up for this one to rename `orFirstParent` to something like `fallbcackToParent`. And [this line ](https://github.com/WordPress/gutenberg/pull/48204/files#diff-d1fbd3af72ee0220c5c3ab2952a24d818174aed711b9a50b57c9544218a63988R1207)is not needed, as you can pass the `selectPrevious` which is `true` at that point.

Reply:
+1. Let's do a quick follow-up PR as Nik proposed and backport changes to the release branch.